components/cluster/pcs/patches/010_man8.patch

```diff
@@ -0,0 +1,94 @@
+--- pcs-0.9.159/pcs/usage.py.orig	2017-09-22 18:54:45.637013891 +0000
```

Comment:
please, use consistent names for patches NUMBER-NAME.patch (not NUMBER_NAME.patch) here and in other places